src/components/Process/Process.tsx

```diff
@@ -0,0 +1,176 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { classNames } from '@shopify/react-utilities/styles';
+import { PROCESS } from '../ThemeIdentifiers';
+import * as baseTheme from './Process.scss';
+
+export interface NavState {
+  indx: number;
+  styles: string[];
+}
+
+export interface Step {
```

Comment:
@HiirenP Add onClick callback as well, make it optional & if user passed it by default return the `event` in callback, so it will be like: `onClick?(returnValue: any): void`

Reply:
But remember this won't work for the inactive state, so if there are 3 steps & 1 step is done & if user tries to click on 3rd step directly then you should not call this callback function